File: VB/UseScaleBreaks/Form1.vb
Imports System
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraCharts

Namespace UseScaleBreaks
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            ' Cast your diagram object to an appropriate diagram type,
            ' to access its axes.
            Dim diagram As XYDiagram = CType(chartControl1.Diagram, XYDiagram)

            ' Enable automatic scale breaks creation, 
            ' and define their maximum number.
            diagram.AxisY.AutoScaleBreaks.Enabled = True
            diagram.AxisY.AutoScaleBreaks.MaxCount = 5

            ' Define the scale breaks' options, which are common
            ' for both manual and automatic scale breaks.
            diagram.AxisY.ScaleBreakOptions.Style = ScaleBreakStyle.Waved
            diagram.AxisY.ScaleBreakOptions.SizeInPixels = 30
            diagram.AxisY.ScaleBreakOptions.Color = Color.Turquoise
        End Sub
    End Class
End Namespace

File: CS/UseScaleBreaks/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraCharts;

namespace UseScaleBreaks {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // Cast your diagram object to an appropriate diagram type,
            // to access its axes.
            XYDiagram diagram = (XYDiagram)chartControl1.Diagram;

            // Enable automatic scale breaks creation, 
            // and define their maximum number.
            diagram.AxisY.AutoScaleBreaks.Enabled = true;
            diagram.AxisY.AutoScaleBreaks.MaxCount = 5;

            // Define the scale breaks' options, which are common
            // for both manual and automatic scale breaks.
            diagram.AxisY.ScaleBreakOptions.Style = ScaleBreakStyle.Waved;
            diagram.AxisY.ScaleBreakOptions.SizeInPixels = 30;
            diagram.AxisY.ScaleBreakOptions.Color = Color.Turquoise;
        }
    }
}
